### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by hunter4466**
Created work cards in HTML and added CSS styles.
Created About section in HTML and added CSS styles.
Created Contact section in HTML and added CSS styles.
No functionalities as javascript ones were added in this step so far.
Checked errors from Linters and fixed them.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -75,5 +75,218 @@ <h2>.</h2>
         <div class="elm_5">.</div>
         <div class="elm_6">.</div>
     </div>
+
+
+    <div class="work_grid">
```
Comment (henatan99) [inline index.html]: - [ ] This should be `section` tag. Always use semantic tags wherever necessary.

File: index.html [hunk 2]
```diff
@@ -75,5 +75,218 @@ <h2>.</h2>
...
+        <div class="box">
```
Comment (henatan99) [inline index.html]: - [  ] This and other `box` class elements should have `li` tags wrapped a parent `ul` tag. Here you are listing your projects.

File: index.html [hunk 3]
```diff
@@ -75,5 +75,218 @@ <h2>.</h2>
...
+            <div class="description item">
+                <p>
+                    A daily selection of privately perzonalized reads; no account or sign-ups required.
+                </p>
+            </div>
+            <div class="languages item">
+                <h5>html</h5>
+                <h5>css</h5>
+                <h5>JavaScript</h5>
+            </div>
```
Comment (henatan99) [inline index.html]: - [  ] I think you are listing the languages you have used in the project you are showcasing. Wouldn't it be greater if you make the items `li` elements wrapped in `ul` tag. Please make similar changes to the other categories too.

File: index.html [hunk 4]
```diff
@@ -75,5 +75,218 @@ <h2>.</h2>
...
+            <div class="button item"><h3><a>See Project</a></h3></div>
+        </div>
+        <div class="box">
+            <div class="prev_5 preview item"></div>
+            <div class="title item">
```
Comment (henatan99) [inline index.html:184]: - [  ] You are begining a new section here. I suggest `section` tag is more appropriate.

File: index.html [hunk 5]
```diff
@@ -75,5 +75,218 @@ <h2>.</h2>
...
+            </div>
+            <div class="description item">
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+            </div>
+            <div class="lets_connect item">
+                <a>LET'S CONNECT</a>
+            </div>
+            <section class="footer item">
```
Comment (henatan99) [inline index.html]: - [  ] This element is kind of heading.  I suggest you replace the `div` tag with an `hx` tag.

File: index.html [hunk 6]
```diff
@@ -75,5 +75,218 @@ <h2>.</h2>
...
+                <h3><a>Get my resume</a></h3>
+            </div>
+            <div class="about_category item">
+                <div>
+                    <h5>Languages</h5>
+                </div>
+                <div class="arrow_op">
+                </div>
+            </div>
+            <div class="lang_prop item">
```
Comment (henatan99) [inline index.html]: - [  ]  The language lists should have the semantically appropriate `|i` wrapped in a `ul` tag. Apply similar changes to the other categories too.

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi there, 

Nice work so far! :clap: 
There are a few changes to make before your project gets approved.

# Changes Requested :recycle:

_See Inline Comments_
OPTIONAL comments are not a requirement for project submission. 

Happy Coding :computer: 

According to Microverse's code review policy you only have a maximum of 3 reviews, a first review and two re-reviews. Make sure you made all the requested changes before you submit for another review.

File: index.html [hunk 7]
```diff
@@ -67,13 +68,237 @@ <h2>.</h2>
...
+            <div class="description item">
+                <p>
+                    A daily selection of privately perzonalized reads; no account or sign-ups required.
+                </p>
+            </div>
+            <ul class="languages item">
+                <li><h5>html</h5></li>
+                <li><h5>css</h5></li>
+                <li><h5>JavaScript</h5></li>
+            </ul>
```
Comment (ElijahTobs) [inline index.html:104]: - [ ] _[Optional]_ In this section, you're required to use the font and colours (background and font) defined for the template. 

Yours | Original
----- | -----
image | image

Comment (ElijahTobs) [inline index.html:104]: - [ ] _[Optional]_ According to the design, each section has a background colour and border radius.

Yours | Original
----- | -----
image | image

**Review by ElijahTobs — APPROVED** [review]
# STATUS: APPROVED 🎉 🥇 
## Hi @hunter4466 , 

Good job making the changes required by the previous Code Reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

- It is advisable that your commit messages be written in the imperative tense, not past tense. You can watch [this video](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6) on how to edit your commit messages using `git rebase`.
Examples of a good commit message:
 i. Make semantic tag fixes on HTML
ii. Fix HTML syntax

_Check for changes marked as Optional in the inline comments below for more Optional changes._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

_Reviewed by_: __**Elijah Ayandokun**__

File: index.html (hunk 1 above)
Comment (ElijahTobs) [reply index.html]: This issue has been addressed ✔️

File: index.html (hunk 2 above)
Comment (ElijahTobs) [reply index.html]: This issue has been addressed ✔️

File: index.html (hunk 3 above)
Comment (ElijahTobs) [reply index.html]: This issue has been addressed ✔️

File: index.html (hunk 4 above)
Comment (ElijahTobs) [reply index.html:184]: This issue has been addressed ✔️

File: index.html (hunk 5 above)
Comment (ElijahTobs) [reply index.html]: This issue has been addressed ✔️

File: index.html (hunk 6 above)
Comment (ElijahTobs) [reply index.html]: This issue has been addressed ✔️

**Pull request merged**